pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.ruff]
+lint.select = [
+  "E",   # pycodestyle
+  "F",   # pyflakes
+  "UP",  # pyupgrade
+  "I",   # organize imports
+  "D",   # docstyle
+]
+
+[tool.ruff.format]
+quote-style = "single"
+
+[tool.ruff.lint.pydocstyle]
+convention = "google"
+
+[tool.ruff.lint.per-file-ignores]
+"exceptions.py" = ["D107"] ## ignore missing doc strings in __init__ for exceptions.
+"tests/**/test_*" = ["D102"] ## ignore missing doc strings in test files.
+
+[[tool.mypy.overrides]]
+module = "harmony_lib.*"
+ignore_missing_imports = true
```

Comment:
I did not add mypy.  I could get rid of this.

Reply:
...or you could add `mypy`. 🧌